### PR TITLE
Fe feature contact us

### PR DESCRIPTION
**Pull request by vnko1**

### CODE REVIEW

File: frontend/src/pages/CardPage/components/index.js [hunk 1]
```diff
@@ -0,0 +1 @@
+export { default } from './Path/Path';
```
Comment (annaBalabak) [inline frontend/src/pages/CardPage/components/index.js:1]: delete repetition

File: frontend/src/pages/ContactUsPage/components/Contacts/Contacts.jsx [hunk 2]
```diff
@@ -0,0 +1,56 @@
...
+
+import { mockContacts } from '../../../../data/catalog';
+
+import styles from './Contacts.module.scss';
+
+const Contacts = ({ href }) => {
+  return (
+    <>
+      <div className={styles.contacts}>
+        <div className={styles['contacts-address']}>
```
Comment (annaBalabak) [inline frontend/src/pages/ContactUsPage/components/Contacts/Contacts.jsx:11]: use dot notation for styles, as we do not use brackets anywhere in a project

File: frontend/src/pages/ContactUsPage/components/GoogleMap/GoogleMap.jsx [hunk 3]
```diff
@@ -0,0 +1,39 @@
...
+          <ICONS.location /> Київ
+        </span>
+      </a>
+      <div className={styles.gMap}>
+        <div className={styles['gMap-canvas']}>
+          <iframe
+            // eslint-disable-next-line max-len
+            src={`https://www.google.com/maps/embed?pb=!1m18!1m12!1m3!1d2540.9623584901133!2d30.54657677695061!3d50.44180178798602!2m3!1f0!2f0!3f0!3m2!1i1024!2i768!4f13.1!3m3!1m2!1s0x40d4cfa592b6919d%3A0xc6cb9d15524f6ff4!2z0YPQuy4g0JjQstCw0L3QsCDQnNCw0LfQtdC_0YssIDExLCDQmtC40LXQsiwgMDIwMDA!5e0!3m2!1sru!2sua!4v1721016674684!5m2!1s${language}!2sua`}
+            className={styles['gMap-iframe']}
+            width="100%"
```
Comment (annaBalabak) [inline frontend/src/pages/ContactUsPage/components/GoogleMap/GoogleMap.jsx:27]: move styling to the styles folder

File: frontend/src/pages/ContactUsPage/components/index.js [hunk 4]
```diff
@@ -0,0 +1,2 @@
+export { default as GoogleMap } from './GoogleMap/GoogleMap';
+export { default as Contacts } from './Contacts/Contacts';
```
Comment (annaBalabak) [inline frontend/src/pages/ContactUsPage/components/index.js:2]: delete repetition

**Review by annaBalabak — CHANGES_REQUESTED** [review]

File: frontend/src/pages/CardPage/components/index.js (hunk 1 above)
Comment (vnko1) [reply frontend/src/pages/CardPage/components/index.js:1]: fixed

File: frontend/src/pages/ContactUsPage/components/Contacts/Contacts.jsx (hunk 2 above)
Comment (vnko1) [reply frontend/src/pages/ContactUsPage/components/Contacts/Contacts.jsx:11]: fixed

File: frontend/src/pages/ContactUsPage/components/index.js (hunk 4 above)
Comment (vnko1) [reply frontend/src/pages/ContactUsPage/components/index.js:2]: fixed

File: frontend/src/pages/ContactUsPage/components/GoogleMap/GoogleMap.jsx (hunk 3 above)
Comment (vnko1) [reply frontend/src/pages/ContactUsPage/components/GoogleMap/GoogleMap.jsx:27]: The width attribute is necessary for the proper rendering of Google Maps in an <iframe>. When Google Maps is embedded in a web page using the <iframe> tag, setting the width of this frame is important for the correct display of the map.

**Review by annaBalabak — APPROVED** [review]

**Pull request merged**